domain/learningandworkprogress/src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/domain/CommonBuilders.kt

```diff
@@ -11,4 +11,17 @@ fun aValidPrisonNumber() = "A1234BC"
 
 fun anotherValidPrisonNumber() = "B5678CD"
 
+fun randomValidPrisonNumber(): String {
```

Comment:
I created this while barking up the wrong tree - thought that I was getting conflicts due to reusing the same prison number, this was not the case but this function may be handy anyway so left it in.